pkg/cmd/ctlstore/main.go

```diff
@@ -102,6 +102,7 @@ type ledgerHealthConfig struct {
 	UnhealthyAttributeValue string        `conf:"unhealth-attribute-value" help:"The value of the attribute if unhealthy"`
 	PollInterval            time.Duration `conf:"poll-interval" help:"How frequently the ledger health should be checked"`
 	AWSRegion               string        `conf:"aws-region" help:"The AWS region to use"`
+	HealthSocket            string        `conf:"health-socket" help:"Path to the health socket file"`
```

Comment:
I wonder if it makes sense to make this more generic. Something like `socket-path`, in case we want to add more behavior to the socket listener down the road.

Reply:
I guess it makes sense in case we end up using the socket for more than just the healthcheck.